plugin/metrics/README.md

```diff
@@ -19,6 +19,7 @@ The following metrics are exported:
 * `coredns_dns_request_type_count_total{server, zone, type}` - counter of queries per zone and type.
 * `coredns_dns_response_size_bytes{server, zone, proto}` - response size in bytes.
 * `coredns_dns_response_rcode_count_total{server, zone, rcode}` - response per zone and rcode.
+* `coredns_plugin_enabled{server, zone, name}` - enabled pulgin(s) per server and zone.
```

Comment:
The plugins are enabled by "ServerBloc", which means per Key at top of each of Stanza.
This Key or ServerBlocKey is available through the line:
```
server := metrics.WithServer(ctx)
```

Why not using this definition - which is used by almost all plugins ?

Reply:
If I understood correctly, you are asking why I need the `zone` label for this metric, I will give an example here:
Here is a `Corefile`:
```
coredns.io:1053 {
    prometheus
    errors
    log
}

example.io:1053 {
    errors
    log
}

coredns.io:53 {
    prometheus
    whoami
    log
}

example.io:53 {
    prometheus
    log
    errors
}
```
In this case, I can get information about which plugins are enabled not only on server but also on the zone:
```
coredns_plugin_enabled{name="errors",server="dns://:1053",zone="coredns.io."} 1
coredns_plugin_enabled{name="errors",server="dns://:53",zone="example.io."} 1
coredns_plugin_enabled{name="log",server="dns://:1053",zone="coredns.io."} 1
coredns_plugin_enabled{name="log",server="dns://:53",zone="coredns.io."} 1
coredns_plugin_enabled{name="log",server="dns://:53",zone="example.io."} 1
coredns_plugin_enabled{name="prometheus",server="dns://:1053",zone="coredns.io."} 1
coredns_plugin_enabled{name="prometheus",server="dns://:53",zone="coredns.io."} 1
coredns_plugin_enabled{name="prometheus",server="dns://:53",zone="example.io."} 1
coredns_plugin_enabled{name="whoami",server="dns://:53",zone="coredns.io."} 1
```

That's why I did this, I just want to get more detailed information from this metric.

But in this case it also has a drawback:
If you want to monitor the whole server, you need to enable `metrics` plugin for every zone of this server(In the above case, we can not see the enabled plugins of the zone="example.io", server="dns://:1053".

To be honest, I don't know which one is better, so maybe you have some ideas?

Reply:
@jiachengxu : my bad !  I had not the correct view of what is in the "server" part (I though was including the zone, but no). 
So I agree with your view and then the way you implemented here.